src/test/unit/views/home.test.ts

```diff
@@ -17,7 +18,9 @@ describe('Home page', () => {
 
   it('should display header',  () => {
```

Comment:
the header test you updated was actually checking for title header not the black bar header on top of the page (nav bar). Probably wording 'should display header' needs refactoring to 'should display title'
I would just add one more test rather  checking Subscriptions tag in the nav bar

Reply:
Absolutely right - changes made